htdocs/public/onlinesign/newonlinesign.php

```diff
@@ -193,6 +193,15 @@
 	}
 
 	$object->fetch(0, $ref);
+	//Trigger propal signe en ligne
```

Comment:
I pushed a commit to add the context on $object so this trigger call is useless.